dashboard/common/utils.py

```diff
@@ -1,14 +1,16 @@
-import logging, os
+import logging, os, HttpRequest
```

Comment:
This `HttpRequest` import is throwing a log error. See comment about `get_myla_globals` below.

Reply:
Hmm, when I remove it mypy says "HttpRequest is not defined".

I think they're from django.http.